src/utils/string.ts

```diff
@@ -0,0 +1,7 @@
+export function stripAnsii(str: string): string {
```

Comment:
l: instead of stripping we could give the `getCopyPasteCodeSnippet` an additional param `colors` that will determine the output. But whatever you prefer!

Reply:
It's a good point. I had something similar in mind but abandoned it because stripping was faster and retained better readability of the snippet in our code 😅 However, it feels kind of like an after thought to the original function and somewhat hacky. So I'll change it.